src/layouts/settings.vue

```
@@ -4,17 +4,20 @@ import { Capacitor } from '@capacitor/core'
-import IconPlans from '~icons/material-symbols/price-change'
-import IconNotification from '~icons/mdi/message-notification'
-import IconPassword from '~icons/mdi/password'
-import IconAccount from '~icons/mdi/user'
-import IconBilling from '~icons/mingcute/bill-fill'
+import IconNotification from '~icons/heroicons/bell'
+import IconBilling from '~icons/heroicons/building-library'
+import IconUsage from '~icons/heroicons/chart-pie'
+import IconPlans from '~icons/heroicons/credit-card'
+import IconCredits from '~icons/heroicons/currency-dollar'
+import IconPassword from '~icons/heroicons/lock-closed'
+import IconAcount from '~icons/heroicons/user'
```

ESLint

[error] 'IconAcount' is defined but never used.

(unused-imports/no-unused-imports)